processing/data_collection/gazette/spiders/pi_teresina.py

```diff
@@ -0,0 +1,49 @@
+import dateparser
+
+from datetime import datetime
+from scrapy import Request, Spider
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class PiTeresinaSpider(BaseGazetteSpider):
+    MUNICIPALITY_ID = '2211001'
+    GAZETTE_URL = 'http://www.dom.teresina.pi.gov.br'
+
+    ATTACHMENT_CSS = 'td:last-child a::attr(href)'
+    DATE_CSS = 'td:nth-child(2)::text'
+    GAZETTE_ROW_CSS = '.table tbody tr'
+    NEXT_PAGE_CSS = '.box-paginacao a.paginacao:last-child::attr(href)'
+    PDF_HREF_CSS = 'td:nth-child(3) a::attr(href)'
+
+    allowed_domains = ['www.dom.teresina.pi.gov.br']
+    name = 'pi_teresina'
+    start_urls = [f'{GAZETTE_URL}/lista_diario.php']
```

Comment:
As you are using `urljoin` to follow the pagination (see my comment below), I don't see the need for `GAZETTE_URL` variable. Just include the full URL in `start_urls` list.

Reply:
Done @rennerocha! 👍